lab5/src/main/java/edu/csula/web/EditEventServlet.java

```diff
@@ -0,0 +1,82 @@
+package edu.csula.web;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Collection;
+import java.util.ArrayList;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import edu.csula.storage.servlet.EventsDAOImpl;
+import edu.csula.storage.EventsDAO;
+import edu.csula.models.Event;
+
+@WebServlet("/admin/editEvent")
+public class EditEventServlet extends HttpServlet {
+
+	@Override
+	public void doGet( HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    EventsDAO dao = new EventsDAOImpl(getServletContext());
+  	Collection<Event> events = dao.getAll();
+		int id = Integer.parseInt(request.getParameter("set_Id"));
+    Event event = null;
+		for (Event e : events){
+			if (e.getId() == id){
+				event = e;
+			}
+		}
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		// TODO: render the events page HTML
+		out.println("<html>");
+		out.println("	<head>");
+		out.println("		<meta charset=\"UTF-8\">");
+		out.println("		<title>Incremental Game</title>");
+		out.println("		<link rel=\"stylesheet\" type=\"text/css\" href=\"../style.css\">");
+		out.println("	</head>");
+		out.println("	<body>");
+		out.println("		<div class=\"main\">");
+		out.println("			<div class=\"header\">");
+		out.println("				<header><h1>Incremental Game Framework</h1></header>");
+		out.println("			</div>");
+		out.println("			<div class = \"flex\">");
+		out.println("				<div class = \"flex-form\">");
+		out.println("					<ul class = \"form\">");
+		out.println("						<a class=\"link_buttons\" href=\"events\">Back</a>");
+		out.println("						<form method = \"POST\" id=\"edit_events\">");
+    out.println("             <h3>Edit Event</h3>");
+		out.println("							<li><label for = \"event_name\">Event Name:</label></li>");
+		out.println("							<li><input type=\"text\" id = \"event_name\" name = \"event_name\" value="+ event.getName() + "></li>");
+		out.println("							<li><label for = \"description\">Event Description</label></li>");
+		out.println("							<li><textarea id = \"description\" name=\"description\">"+ event.getDescription() +"</textarea></li>");
+		out.println("							<li><label for = \"trigger\">Trigger at</label></li>");
+		out.println("							<li><input type=\"number\" id=\"trigger\" name=\"trigger\" value="+ event.getTriggerAt() + "></li>");
+		out.println("							<li><button>Add</button></li>");
+		out.println("						</form>");
+		out.println("					</ul>");
+		out.println("				</div>");
+		out.println("			</div>");
+		out.println("		</div>");
+		out.println("	</body>");
+		out.println("</html>");
```

Comment:
edit is not yet refactor to JSP

Reply:
Sorry my mistake, i didn't pay attention on the SRS due to simultaneously working on lab4, lab5, homework2 and homework3 and got confusesd. I did it in homework 3 by mistake. Can i change it now because i don't want to lose my marks on my silly mistake.

Reply:
Eric the SRS never mentioned anything about the EditEventServlet being in JSP. But i did it in homework 3.